config/test.exs

```diff
@@ -3,3 +3,5 @@ import Config
 config :logger, level: :warning
 
 config :testcontainers, log_level: :warning
+
+config :kafka_ex, disable_default_worker: true
```

Comment:
this must also be added to config/config.exs. Currently verify version fails because that command isnt run in test environment. Any config we place in config/config.exs shouldn't propagate to the using application anyway.